README.md

```diff
@@ -88,6 +89,32 @@ print("average: $(mean(m.pixels))\n")
 
 See the document [CONTRIBUTING.md](https://github.com/ziotom78/Healpix.jl/blob/master/CONTRIBUTING.md).
 
+## Citing Healpix.jl
+
+If you have used Healpix.jl in your research and want to acknowledge the library in your academic publications, you can reference its  [ASCL entry](http://ascl.net/code/v/3025) through the code `ascl:2109.028`:
+
+> Tomasi M., Li Z. 2021 Healpix.jl: Julia-only port of the HEALPix library, 0.30, Astrophysics Source Code Library ascl:2109.028
+
+Here is a BibTeX entry ready to be used, generated by [ADS](https://ui.adsabs.harvard.edu/abs/2021ascl.soft09028T/abstract):
+
+```
+@MISC{2021ascl.soft09028T,
+  author = {{Tomasi}, M. and {Li}, Z.},
+  title = "{Healpix.jl: Julia-only port of the HEALPix library}",
+  keywords = {Software},
+  howpublished = {Astrophysics Source Code Library},
+  year = 2021,
+  month = oct,
+  archivePrefix = "ascl",
```

Comment:
I think an entry `eprint = {2109.028},` is missing.

Reply:
Done, thanks!